dvc/fs/__init__.py

```diff
@@ -19,13 +21,26 @@
     ConfigError,
     RemoteMissingDepsError,
 )
-from dvc_objects.fs.implementations.local import localfs  # noqa: F401
 from dvc_objects.fs.path import Path  # noqa: F401
 
 from .data import DataFileSystem  # noqa: F401
 from .dvc import DvcFileSystem  # noqa: F401
 from .git import GitFileSystem  # noqa: F401
 
+known_implementations.update(
+    {
+        "dvc": {
+            "class": "dvc.fs.dvc.DvcFileSystem",
+            "err": "dvc is supported, but requires 'dvc' to be installed",
+        },
+        "git": {
+            "class": "dvc.fs.git.GitFileSystem",
+            "err": "git is supported, but requires 'dvc' to be installed",
+        },
+    }
+)
```

Comment:
For the record, we'll register other plugins here the same way in the near future.